aws/tagsWAF.go

```diff
@@ -0,0 +1,118 @@
+package aws
```

Comment:
Hi @DrFaust92 👋 Thank you for working on this! Your timing with this contribution lines up well with some improvements we've implemented within the codebase to prevent the need to manually create these tagging files and functions. The Contribution Guide update is in review here if you would like to read up on this (https://github.com/terraform-providers/terraform-provider-aws/pull/10313/files) and the `keyvaluetags` library is already in place with support for WAF tagging. 🎉 

We can remove this `tagsWAF.go` and its testing file -- I'll try to point to specific code updates that can use the new `keyvaluetags` library functionality. 👍

Reply:
thanks @bflad , i was just thinking that i had to do a lot of duplication from other resources for this!